contacts/index_settings.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
bigger goal here is moving all contact specific stuff into this package and making it easier to add support in future for indexing of other things - messages, tickets etc